### ISSUE TITLE
hello help please

### ISSUE THREAD

**Issue by sam1991a** [BenBurak/Textdraw-Streamer#31]
Hello, I am having problems with a message that is printed in the log and creates spam. How can I stop the flood from causing lag on the server?

[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.
[textdraw.streamer] DynamicPlayerTextDrawShow: First create the player textdraw.

**Comment by nexquery** [comment]
Use the latest version of the plugin and see where the problem is coming from. (file:line)

**Comment by sam1991a** [comment]
> Use the latest version of the plugin and see where the problem is coming from. (file:line)


The last version was 2.0.2?

**Comment by nexquery** [comment]
No, [v2.0.3](https://github.com/nexquery/samp-textdraw-streamer/releases/tag/v2.0.3-hotfix)

**Comment by sam1991a** [comment]
update plugin and compile and I get the error undefined symbol "__file" help please

**Comment by nexquery** [comment]
Use latest version compiler: https://github.com/pawn-lang/compiler/releases

**Comment by sam1991a** [comment]
Hello, sorry for the delay in answering, I haven't been able to identify the error. Would there be another way to know where the error comes from? If you don't mind, would you like to show my code?

**Comment by nexquery** [comment]
You need to use the latest version of the plugin and the latest version of the compiler to detect the problem.
